pubpub/pubpub
# feat: put custom scripts behind feature flag

## Issue(s) Resolved

N/A

## Test Plan

1. Visit a community with the `"customScripts"` feature flag enabled (e.g. https://demo.duqduq.org). You should be able to add custom JS via the dashboard.
2. Visit a community without the `"customScripts"` feature flag enabled. You should not be able to add custom JS.
